packages/react-table/src/components/Table/utils/decorators/collapsible.tsx

```diff
@@ -56,12 +56,51 @@ export const expandedRow = (colSpan: number) => {
         extraParams: { contentId = 'expanded-content' }
       }
     }: IExtra
-  ): decoratorReturnType =>
-    rowData.hasOwnProperty('parent') && {
-      // todo: rewrite this logic, it is not type safe
-      colSpan: colSpan + (!!rowData.fullWidth as any),
-      id: contentId + rowIndex,
-      className: rowData.noPadding && css(styles.modifiers.noPadding)
-    };
+  ): decoratorReturnType | false => {
```

Comment:
Can we change this to `Nullable<decoratorReturnType>` instead of `decoratorReturnType | false` and return in the end `null` instead of `false`?

Reply:
Is there any  `Nullable` type ? else I can just use `decoratorReturnType | null` ? 
I only see `NonNullable`.

Reply:
Or maybe you would prefer if I add `export type Nullable<T> = T | null;` to  [typeUtils](https://github.com/patternfly/patternfly-react/blob/2210ec2c142a385f85de8b0bac9c61a61521186d/packages/react-core/src/helpers/typeUtils.ts)?

Reply:
I think `decoratorReturnType | null` will work fine as a type. We do this many other places in PatternFly.